### PR TITLE
P0 badge and button

### PR DESCRIPTION
**Pull request by sumitk-singh**
waiting for reviews

### CODE REVIEW

File: components/Badge.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    padding-top: 3.125rem;
+    align-items: center;
+}
+
+
+/* for m size */
+
+.status-m-size {
+    border: 0.125rem solid white;
+    border-radius: 50%;
```
Comment (srishtigautam21) [inline components/Badge.css:37]: you can use px for such small values

File: components/Badge.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    margin-left: 2.4rem;
+    position: relative;
+}
+
+
+/* *** */
+
+.online {
+    background: green;
+}
```
Comment (srishtigautam21) [inline components/Badge.css:66]: use var for colors

**Review by srishtigautam21 — COMMENTED** [review]
i have suggested some small changes

File: components/Button.css [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+
+.floating-btn {
+    cursor: pointer;
+    background: black;
+    color: white;
+    padding: 12px;
+    height: 20px;
+    width: 20px;
+    border-radius: 50%;
+    /* padding: 0.5rem 1rem; */
```
Comment (DivyanshSareen) [inline components/Button.css]: remove commented code

File: components/Button.css [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+    cursor: pointer;
+}
+
+.floating-btn {
+    cursor: pointer;
+    background: black;
+    color: white;
+    padding: 12px;
+    height: 20px;
+    width: 20px;
```
Comment (DivyanshSareen) [inline components/Button.css]: use relative units

File: components/Button.css (hunk 3 above)
Comment (archit7448) [inline components/Button.css]: you can remove commented code

File: components/Badge.css (hunk 2 above)
Comment (archit7448) [reply components/Badge.css:66]: +1

File: components/Badge.css (hunk 1 above)
Comment (archit7448) [reply components/Badge.css:37]: +1

**Review by DivyanshSareen — COMMENTED** [review]
The components and doc site look great!

File: docs-page.html [hunk 5]
```diff
@@ -144,10 +145,178 @@ <h2>Example of Avatar</h2>
...
+                </h3>
+                <!-- primary solid button -->
+                <h2>Example of Solid Button Styles</h2>
+                <p>
+                    Whenever you want your user to have CTA (call to action) on a link or button, use primary style buttons.
+                </p>
+                <p>As you want to use success button, then use the class accordingly. (For e.g <span class="highlight-text">class="btn solid-success"</span></p>
+                <p>Check code below to copy the html part as follow below.</p>
+                <div class="solid-buttons-example">
+                    <a class="btn solid-primary">Primary</a>
```
Comment (itsmebipul00) [inline docs-page.html]: links are designed as buttons here i guess, buttons are for some action(with routing sometimes) and links are only for routing

File: docs-page.html [hunk 6]
```diff
@@ -144,10 +145,178 @@ <h2>Example of Avatar</h2>
...
+                </div>
+                <!-- end of primary solid button -->
+
+                <!-- outline primary button -->
+                <h2>Example of Solid Button Styles</h2>
+                <p>
+                    you may also use outline button for CTA as per the required design.
+                </p>
+                <div class="outline-buttons-example">
+                    <a class="btn outline-primary">Primary</a>
```
Comment (itsmebipul00) [inline docs-page.html]: Same here

**Comment by itsmebipul00** [comment]
Small changes required, all rest good to go

File: components/Badge.css (hunk 1 above)
Comment (sumitk-singh) [reply components/Badge.css:37]: ok

File: components/Button.css (hunk 3 above)
Comment (sumitk-singh) [reply components/Button.css]: done

Comment (sumitk-singh) [reply components/Button.css]: done

File: components/Badge.css (hunk 2 above)
Comment (sumitk-singh) [reply components/Badge.css:66]: done

File: components/Badge.css (hunk 1 above)
Comment (sumitk-singh) [reply components/Badge.css:37]: done

**Review by Ankur9669 — APPROVED** [review]
image
Apart from the changes that are already told by peers, i would just add that you could properly indent the code in the iframe.

This UI looks awesome, so great work!!!

**Review by srishtigautam21 — APPROVED** [review]

**Pull request merged**